payara-managed/src/main/java/fish/payara/arquillian/container/payara/managed/PayaraManagedContainerConfiguration.java

```diff
@@ -162,8 +139,10 @@ public String getTarget() {
      * Validates if current configuration is valid, that is if all required properties are set and
      * have correct values
      */
+    @Override
     public void validate() throws ConfigurationException {
-        notNull(getGlassFishHome(), "The property glassFishHome must be specified or the GLASSFISH_HOME environment variable must be set");
+        notNull(getGlassFishHome(), String.format("The property glassFishHome must be specified or "
+                + "the %s environment variable must be set", PAYARA_HOME_PROPERTY));
```

Comment:
`PAYARA_HOME_PROPERTY` value is fetched via `System.getProperty(PAYARA_HOME_PROPERTY)` but validate function sends environment variable hint.
IMHO we should support GLASSFISH_HOME env var as deprecated config and remove later in major version release, wdyt?

Reply:
> IMHO we should support GLASSFISH_HOME env var as deprecated config 

I wouldn't think so b/c this is not an external facing variable, i.e. not documented anywhere and not guaranteed to be compatible with anything else.

I know I probably jumped the gun, but this PR is released and already in maven central and should be merged as is.

Reply:
I will issue another PR to 2.4-SNAPSHOT to fix the comment.

Reply:
It is used in examples for [Glassfish](https://itnext.io/testing-jakarta-ee-9-applications-with-arquillian-and-glassfish-v6-9981e454e6eb), I'm not sure if this is a good step, especially if we don't document it anywhere.

Reply:
Well perhaps we should document t it. This is Payara not GlassFish. GlassFish has its own connectir